src/corePlugins.js

```diff
@@ -631,14 +631,12 @@ export let corePlugins = {
         ['inset-x', ['left', 'right']],
         ['inset-y', ['top', 'bottom']],
       ],
-      [
-        ['start', ['inset-inline-start']],
-        ['end', ['inset-inline-end']],
-        ['top', ['top']],
-        ['right', ['right']],
-        ['bottom', ['bottom']],
-        ['left', ['left']],
-      ],
+      [['top', ['top']]],
+      [['right', ['right']]],
+      [['end', ['inset-inline-end']]],
+      [['bottom', ['bottom']]],
+      [['left', ['left']]],
+      [['start', ['inset-inline-start']]],
```

Comment:
Do you think we should keep `start` and `end` as-is? This change would sort it `end` first then `start` which seems odd? 🤔

Reply:
Good question. Yo @adamwathan what do you think?

Reply:
Yeah I put them there on purpose but doing some more research, it looks like the logical ones will be flipped when/if that’s ever supported:

![image](https://user-images.githubusercontent.com/4323180/229933417-1a4f570a-3ca0-4a78-8646-86b3bd45c92f.jpeg)

So we should go top, right, inline-start, bottom, left, inline-end I think.